docs/userGuide/components/others.md

```diff
@@ -18,7 +18,7 @@
 
 <div id="overview" class="lead">
 
-This page lists some other components that may be useful in creating education websites.
+This page lists some other components that may be useful in creating education websites. For now, there are only question and quiz components.
```

Comment:
I know that we removed this sentence in #2206 so it is being added back, but I am not sure if this sentence is really necessary 😅